src/main/app/controller/GetController.ts

```diff
@@ -22,7 +22,11 @@ export type TranslationFn = ({
 
 @autobind
 export class GetController {
-  constructor(protected readonly view: string, protected readonly content: TranslationFn | Translations) {}
+  constructor(
+    protected readonly view: string,
+    protected readonly content: TranslationFn | Translations,
+    protected language = 'en'
```

Comment:
I don't see how we can set this at boot time.

Reply:
👍 Good spot, I've updated this in 6b56ffd to make it clearer than `'en'` is being used as the default and then this gets updated there's a language set in the session. The session for these sign out pages will be destroyed, so this allows the child to update the parents language before it gets destroyed.

Reply:
👍 Good spot, I've changed this so that pages with a "Save and sign out" link use the `?lng=` query string to keep their preference on these pages that clear out of the users session.